topics/metagenomics/tutorials/mothur-miseq-sop/tours/tour.yaml

```diff
@@ -0,0 +1,1984 @@
+  - title: Step 125. Venn diagram
+    element: '#tool-search'
+    content: |-
+      Execute the tool with  <ul>
+        <li>Set OTU Shared to Subsample.shared file from previous step</li>
+        <li>Set groups to F3D0,F3D1,F3D2,F3D3</li>
+      </ul>
+    position: left
+  - title: Step 126. Venn diagram
+    content: >-
+      Observe the result. You should see that  there were a total of 180 OTUs
+      observed between the 4 time points. Only 76 of those OTUs were shared by
+      all four time points. We could look deeper at the shared file to see
+      whether those OTUs were umerically rare or just had a low incidence.
+    backdrop: false
+  - title: Tree
+    content: >-
+      Next, let’s generate a dendrogram to describe the similarity of the
+      samples to each other. We will generate a dendrogram using the jclass and
+      thetayc calculators within the tree.shared command
+    backdrop: true
+  - title: Step 127. Tree
+    element: '#tool-search-query'
+    content: Search for Tree.shared tool
+    placement: right
+    textinsert: Tree.shared
+  - title: Step 128. Tree
+    element: '#tool-search'
+    content: Click on the "Tree.shared" tool to open it
+    placement: right
+    postclick:
+      - >-
+        a[href$="/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_tree_shared%2Fmothur_tree_shared%2F1.36.1.0"]
+  - title: Step 129. Tree
+    element: '#tool-search'
+    content: |-
+      Execute the tool with  <ul>
+        <li>“Select input format” to Phylip Distance Matrix</li>
+        <li>“phylip” to dist files from Dist.shared (collection)</li>
+      </ul>
+    position: left
+  - title: Step 130. Tree
+    element: '#tool-search-query'
+    content: Search for Newick display tool
+    placement: right
+    textinsert: Newick display
+  - title: Step 131. Tree
+    element: '#tool-search'
+    content: Click on the "Newick display" tool to open it
+    placement: right
+    postclick:
+      - >-
+        a[href$="/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdcorreia%2Fnewick_display%2Fnwdisplay%2F1"]
+  - title: Step 132. Tree
+    element: '#tool-search'
+    content: |-
+      Execute the tool with  <ul>
+        <li>“Newick file” to output from Tree.shared (collection)</li>
+      </ul>
+    position: left
+  - title: Step 133. Tree
+    element: >-
+      #current-history-panel .controls .actions a[href$="javascript:void(0);"]
+      .fa.fa-check-square-o
+    content: >-
+      Inspect the output. It shows that the early and late communities cluster
+      with themselves to the exclusion of the others
+  - title: Determine statistical significance of clusterings
+    content: >-
+      We can perform a test to determine whether the clustering within the tree
+      is statistically significant or not using by choosing from the parsimony,
+      unifrac.unweighted, or unifrac.weighted commands. To run these we will
+      first need to crea
```

Comment:
fix closing tag

Reply:
huh, wrong again? well, this but at line 1886 (in one the classify.rf steps)

Reply:
@shiltemann thanks for the comments, I will fix :)